app-java/src/main/java/ly/count/java/demo/BackendModeExample.java

```diff
@@ -0,0 +1,180 @@
+package ly.count.java.demo;
+
+import ly.count.sdk.java.Config;
+import ly.count.sdk.java.Countly;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class BackendModeExample {
+    final static String DEVICE_ID = "device-id";
+    final static String COUNTLY_APP_KEY = "YOUR_APP_KEY";
+    final static String COUNTLY_SERVER_URL = "https://try.count.ly/";
+
+    static void recordEvent() {
+        Map<String, Object> segment = new HashMap<String, Object>() {{
+            put("Time Spent", 60);
+            put("Retry Attempts", 60);
+        }};
+
+        Countly.backendMode().recordEvent(DEVICE_ID, "Event Key", 1, 0, 5, segment, 0);
+    }
+
+    static void setLocation() {
+        Countly.api().addLocation(31.5204, 74.3587);
+    }
+
+    static void recordUserProperties() {
```

Comment:
this should create only a single map

Reply:
how could I record the operations(inc, multipy) on the property by using a single map?
![image](https://user-images.githubusercontent.com/8924406/159909018-9f94595e-c15b-4d17-a6af-af24973942df.png)